accord-core/src/main/java/accord/local/Command.java

```diff
@@ -301,28 +302,27 @@ public AcceptOutcome accept(SafeCommandStore safeStore, Ballot ballot, PartialRo
         }
 
         TxnId txnId = txnId();
-        KeyRanges coordinateRanges = coordinateRanges(safeStore);
-        KeyRanges executeRanges = txnId.epoch == executeAt.epoch ? coordinateRanges : safeStore.ranges().at(executeAt.epoch);
-        KeyRanges acceptRanges = txnId.epoch == executeAt.epoch ? coordinateRanges : safeStore.ranges().between(txnId.epoch, executeAt.epoch);
+        Ranges coordinateRanges = coordinateRanges(safeStore);
+//        Ranges executeRanges = txnId.epoch == executeAt.epoch ? coordinateRanges : safeStore.ranges().at(executeAt.epoch);
```

Comment:
Remove `executeRanges`? Why don't we use the ranges at the `executeAt.epoch` anymore?

Reply:
We don't need to adopt any information about a proposed execution epoch, only if we actually decide to use that execution epoch. For recovery only the coordination epoch matters, so this was I think cleaning up a little the earlier changes we made to stop requiring preaccept prior to accept